app/views/schools/contact_us/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <% self.page_title = "Contact us" %>
-
+<% content_for :back_link do govuk_back_link(schools_dashboard_path); end %>
```

Comment:
I don't agree with this @MylesJarvis - the Back link should be used for multi-page transactions, rather than just to take people to the home page?
https://design-system.service.gov.uk/components/back-link/

I appreciate that this is used in production on other pages (eg Accessibility statement, privacy policy, etc.) - I don't think it should be

Suggest taking UCD advice!

Reply:
I think the issue is that the MSE breadcrumb doesn't include `Home`? Should it? And then it would work on pages like these

Rather than pushing out this change, I think it would be better to take a step back and review this (but it's a very low priority!)

Reply:
This change needs removing?

Reply:
Design system does recommend that the breadcrumb goes back to a 'Home'. In this instance we've badged the school dashboard as 'home' and used the school name. I think that's an appropriate use based on my reading! I'm going to mark this as resolved.